src/actions/channels.js

```diff
@@ -79,6 +80,7 @@ export function createChannel(channel, userId) {
 
         if (!myMembers[created.id]) {
             actions.push({type: ChannelTypes.RECEIVED_MY_CHANNEL_MEMBER, data: member});
+            loadRolesIfNeeded(new Set(member.roles.split(' ')))(dispatch, getState);
```

Comment:
Can we change these to 
`dispatch(loadRolesIfNeeded(new Set(member.roles.split(' '))));`

That way we don’t care if the action is a thunk or not

Reply:
Done